src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -172,13 +182,23 @@
         <note>key: error.page.404.actions</note>
       </trans-unit>
       <trans-unit id="7027b2adcf028b82111334763e40823cd85a52fa" resname="error.page.404.message">
-        <source><![CDATA[Error 404: the page you requested was not found.<br/>It might have been deleted or it is temporarily unavailable.]]></source>
-        <target state="new"><![CDATA[Error 404: the page you requested was not found.<br/>It might have been deleted or it is temporarily unavailable.]]></target>
+        <source><![CDATA[The page you requested was not found.<br/> It might have been deleted or it is temporarily unavailable.]]></source>
+        <target state="new"><![CDATA[The page you requested was not found.<br/> It might have been deleted or it is temporarily unavailable.]]></target>
         <note>key: error.page.404.message</note>
       </trans-unit>
+      <trans-unit id="7596368af317fe78b955e2a3f63c7586900bcd76" resname="error.page.404.messagea">
+        <source>The page you requested was not found. It might have been deleted or it is temporarily unavailable.</source>
+        <target state="new">The page you requested was not found. It might have been deleted or it is temporarily unavailable.</target>
+        <note>key: error.page.404.messagea</note>
+      </trans-unit>
+      <trans-unit id="ffa7f20226d38a989f6c931264bd4cf485b435e6" resname="error.page.404.sub_title">
+        <source>You probably weren't expecting to land here.</source>
```

Comment:
Do we even need this? It's 404, it's safe to assume nobody wants to land there.

Reply:
Yes, we do need it. It is a standard. We are also aligning error views in Ibexa site and here.